XmlValidator/callbacks.py

```diff
@@ -87,6 +87,9 @@ def is_number(element):
             text = child.text if child.text is not None else ''
             if text[-3:] == '...':
                 text = text[:-3]
+            if '|' in text:
+                index = text.find('|')
+                text = text[:index] + text[index+1:]
```

Comment:
Yip, could have sworn I neatened that one up to the suggested line but must have missed it